Kitodo/src/test/java/de/sub/goobi/helper/encryption/MD4Test.java

```diff
@@ -13,19 +13,18 @@
 
 import static org.junit.Assert.assertTrue;
 
-import java.io.UnsupportedEncodingException;
 import java.util.Arrays;
 import java.util.HashMap;
 
-import org.junit.Test;
+import org.bouncycastle.jce.provider.JDKMessageDigest.MD4;
 
-import edu.sysu.virgoftp.ftp.encrypt.MD4;
+import org.junit.Test;
 
 public class MD4Test {
     static HashMap<String, byte[]> testData;
 
     static {
-        testData = new HashMap<String, byte[]>();
+        testData = new HashMap<>();
```

Comment:
It is not necessary to write type of key and value in this HashMap. It is already given in line 24.